### PR TITLE
fix(admin): Add comment reports

### PR DESCRIPTION
**Pull request by jhechtf**
Re-adds comment report rendering, which was previously removed.
## Description
Makes some modifications to the admin package in order to re-add rendering for comment reports.

## Related Issue

N/A

## Motivation and Context

We need to be able to see the comments when viewing the comment reports
## How Has This Been Tested?

Locally, clicking through stuff to make sure.

## Screenshots/Video (if applicable):
image

### CODE REVIEW

File: apps/admin/src/app/challenge/[challengeId]/challenge-review.tsx [hunk 1]
```diff
@@ -27,7 +27,7 @@ export function ChallengeReview({ challenge }: Props) {
         variant: 'success',
         title: 'Challenge Approved Successfully',
       });
-      router.push('/admin');
+      router.push('/');
```
Comment (bautistaaa) [inline apps/admin/src/app/challenge/[challengeId]/challenge-review.tsx:30]: do we need to append a querystring so it chooses the right tab?

File: apps/admin/src/app/report/[id]/actions.tsx [hunk 2]
```diff
@@ -59,7 +59,7 @@ export function ReportActions({ report }: ReportActionsProps) {
         title: 'Success',
         description: <p>Comment deleted successfully</p>,
       });
-      router.push('/admin');
+      router.push('/');
```
Comment (bautistaaa) [inline apps/admin/src/app/report/[id]/actions.tsx:62]: do we need to append querystrings for correct tab?

File: packages/ui/src/index.ts [hunk 3]
```diff
@@ -7,6 +7,8 @@ export * from './components/button';
 export * from './components/card';
 export * from './components/checkbox';
 export * from './components/dialog';
+export * from './components/data-table';
```
Comment (bautistaaa) [inline packages/ui/src/index.ts:10]: we removed this on purpose. are you able to import directly?

Comment (jhechtf) [reply packages/ui/src/index.ts:10]: It wasn't building when i tried to import it directly

File: apps/admin/src/app/report/[id]/actions.tsx (hunk 2 above)
Comment (jhechtf) [reply apps/admin/src/app/report/[id]/actions.tsx:62]: mm probably.

**Comment by bautistaaa** [comment]
rebase, barrel file is now deleted

File: apps/admin/src/app/report/[id]/actions.tsx (hunk 2 above)
Comment (bautistaaa) [reply apps/admin/src/app/report/[id]/actions.tsx:62]: looks like so 

```ts
              <Link href="/?tab=tracks">
                <Button variant="ghost">Cancel</Button>
              </Link>
```

**Review by bautistaaa — APPROVED** [review]

**Pull request merged**